tests/integration/schema/updates/with_schema_branch_test.go

```diff
@@ -543,3 +543,63 @@ func TestSchemaUpdates_WithBranchingSchemaAndSetActiveSchemaToOtherBranchThenPat
 	}
 	testUtils.ExecuteTestCase(t, test)
 }
+
+func TestSchemaUpdates_WithBranchingSchemaAndGetCollectionAtVersion(t *testing.T) {
+	schemaVersion2ID := "bafkreidn4f3i52756wevi3sfpbqzijgy6v24zh565pmvtmpqr4ou52v2q4"
+	schemaVersion3ID := "bafkreieilqyv4bydakul5tbikpysmzwhzvxdau4twcny5n46zvxhkv7oli"
+
+	test := testUtils.TestCase{
+		Description: `Test schema update, with branching schema toggling between branches and gets the 
+collection at a specific version`,
+		Actions: []any{
+			testUtils.SchemaUpdate{
+				Schema: `
+					type Users {
+						name: String
+					}
+				`,
+			},
+			testUtils.SchemaPatch{
+				// The second schema version will not be set as the active version, leaving the initial version active
+				SetAsDefaultVersion: immutable.Some(false),
+				Patch: `
+					[
+						{ "op": "add", "path": "/Users/Fields/-", "value": {"Name": "email", "Kind": 11} }
+					]
+				`,
+			},
+			testUtils.SchemaPatch{
```

Comment:
suggestion: 3 schema versions are not required to test this - you could remove the first patch and the `SetActiveSchemaVersion` action and then query for `schemaVersion1ID`

Reply:
Thanks for pointing that out. I got lazy on that copy pasting 😅

Reply:
✅